src/meta/meta_bulk_load_service.cpp

```diff
@@ -1349,9 +1349,197 @@ void bulk_load_service::do_sync_partition(const gpid &pid, std::string &partitio
     });
 }
 
+// ThreadPool: THREAD_POOL_META_SERVER
 void bulk_load_service::try_to_continue_bulk_load()
+{
+    FAIL_POINT_INJECT_F("meta_try_to_continue_bulk_load", [](dsn::string_view) {});
+    for (const auto app_id : _bulk_load_app_id) {
+        app_bulk_load_info ainfo = _app_bulk_load_info[app_id];
```

Comment:
Do we need to judge whether the app_id is contained in _app_bulk_load_info or not?

Reply:
`_bulk_load_app_id` and `_app_bulk_load_info` are sync from remote storage at the same time, you can see function `do_sync_app`.